teachopencadd/talktorials/019_md_simulation/talktorial.ipynb

```diff
@@ -0,0 +1,1250 @@
+{

```

Comment:
<p>Are you sure you need that +1 in <code>complex_positions[len(protein.positions) + 1: total_atoms]</code>?  Python indexing is <code>[start, end)</code>, so I would assume that we can start the 2nd range where the 1st ended, not shifted +1.  What's the extra atom? You can also drop <code>0</code>  and <code>total_atoms</code>.  In other words:</p><p> </p><pre class="ql-syntax" spellcheck="false">&nbsp; &nbsp; complex_positions = unit.Quantity(np.zeros([total_atoms, 3]), unit=unit.nanometers)
&nbsp; &nbsp; complex_positions[:len(protein.positions)] = protein.positions&nbsp; # add protein positions
&nbsp; &nbsp; complex_positions[len(protein.positions):] = ligand.positions&nbsp; # add ligand positions
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/TeachOpenCADD/pull/56/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>